### PR TITLE
258: Wait for PostgreSQL before starting Odoo

### PR DESCRIPTION
**Pull request by rbuisson**
See issue https://github.com/odoo/docker/issues/258

### CODE REVIEW

**Comment by rbuisson** [comment]
In Debian Stretch (Odoo 11 and 12) the **wait-for-it** script is available as a package. This it just needs to be installed via `apt install`.
In Debian Jessie (Odoo 10) however, it needs to be manually provided to the system. I have added this script as a resource and copied it via Dockerfile.

**Comment by talnetd** [comment]
This is a good find. I have been aching for this wait-for-it down to TCP level.

**Comment by rbuisson** [comment]
@d-fence , do you think the above makes sense?

**Comment by rbuisson** [comment]
Hi @d-fence @rakete, what would be the process to get approval on this PR?
(Or any PR in general? Is there a IRC/Slack channel where a contributor can reach out to you?)

**Comment by rbuisson** [comment]
Just updated the PR to add Odoo 13 and remove 10.
(@d-fence )

File: 13.0/entrypoint.sh [hunk 1]
```diff
@@ -23,6 +23,9 @@ check_config "db_port" "$PORT"
 check_config "db_user" "$USER"
 check_config "db_password" "$PASSWORD"
 
+# Wait for the database to be up
+/wait-for-it.sh $HOST:$PORT --timeout=60 -- sleep 5s
```
[the remark this answers is not in the document]
Comment (rbuisson) [reply 13.0/entrypoint.sh]: Done.

**Comment by rbuisson** [comment]
Should be good now. @d-fence

**Pull request closed without merging**